adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -201,18 +201,16 @@ private void handleEndpointConnection(final MqttEndpoint endpoint) {
                     MqttUsernamePassword authObject = MqttUsernamePassword.create(endpoint,
```

Comment:
@sysexcontrol understood. So it's better keeping the `MqttUsernamePassword` name but having username and passoword as parameters getting rid of the `MqttEndpoint`